.github/workflows/tests.yaml

```
@@ -0,0 +1,20 @@
+name: Tests
+on:
+  push:
+    branches:
+      - main
+      - "release/*"
+  pull_request:
+jobs:
+  unit:
+    name: Unit
+    runs-on: 'ubuntu-22.04'
```

Can we use latest in here?
```suggestion
    runs-on: ubuntu-latest
```